services/dy-dash/ans-model-viewer/.cookiecutterrc

```diff
@@ -0,0 +1,38 @@
+# This file exists so you can easily regenerate your project.
+#
+# `cookiepatcher` is a convenient shim around `cookiecutter`
+# for regenerating projects (it will generate a .cookiecutterrc
+# automatically for any template). To use it:
+#
+#    pip install cookiepatcher
+#    cookiepatcher gh:itisfoundation/cookiecutter-osparc-service project-path
+#
+# See:
+#    https://pypi.python.org/pypi/cookiepatcher
+#
+# Alternatively, you can run:
+#
+#    cookiecutter --overwrite-if-exists --config-file=project-path/.cookiecutterrc gh:itisfoundation/cookiecutter-osparc-service
+#
+
+default_context:
+
+    _output_dir:               '/home/iavarone/repos/osparc-services/services/dy-dash'
+    _template:                 'gh:elisabettai/cookiecutter-osparc-service'
+    author_affiliation:        "IT'IS Foundation"
+    author_email:              'iavarone@itis.swiss'
```

Comment:
if you wish to not put your itis address in these repositories. you can use github anonymised email address.
you can find it in your profile settings, under emails. it is following the pattern of XXX@@users.noreply.github.com